start_dev.sh

```
@@ -1,24 +1,44 @@
 #!/bin/bash
 
+# SolSniperX Portable Start Script
+
+# Get the directory where this script is located
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# 1. Start Backend
 echo "Starting SolSniperX Backend..."
-cd /home/mulky/Desktop/SolSniperX/backend
+cd "$SCRIPT_DIR/backend"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)